dspace-api/src/main/java/org/dspace/core/AbstractHibernateDAO.java

```diff
@@ -105,127 +106,87 @@ public T findByID(Context context, Class clazz, int id) throws SQLException {
      */
     public List<T> findMany(Context context, Query query) throws SQLException {
         @SuppressWarnings("unchecked")
-        List<T> result = (List<T>) query.list();
+        List<T> result = (List<T>) query.getResultList();
         return result;
     }
 
-    /**
-     * Create criteria matching an entity type or a supertype thereof.
-     * Use when building a criteria query.
-     *
-     * @param context         current DSpace context.
-     * @param persistentClass specifies the type to be matched by the criteria.
-     * @return criteria concerning the type to be found.
-     * @throws SQLException passed through.
-     */
-    public Criteria createCriteria(Context context, Class<T> persistentClass) throws SQLException {
-        return getHibernateSession(context).createCriteria(persistentClass);
-    }
-
-    /**
-     * Create criteria matching an entity type or a supertype thereof.
-     * Use when building a criteria query.
-     *
-     * @param context         current DSpace context.
-     * @param persistentClass specifies the type to be matched by the criteria.
-     * @param alias           alias for the type.
-     * @return criteria concerning the type to be found.
-     * @throws SQLException passed through.
-     */
-    public Criteria createCriteria(Context context, Class<T> persistentClass, String alias) throws SQLException {
-        return getHibernateSession(context).createCriteria(persistentClass, alias);
-    }
+//    public Criteria createCriteria(Context context, Class<T> persistentClass) throws SQLException {
+//        return getHibernateSession(context).createCriteria(persistentClass);
+//    }
+//
+//    public Criteria createCriteria(Context context, Class<T> persistentClass, String alias) throws SQLException {
+//        return getHibernateSession(context).createCriteria(persistentClass, alias);
+//    }
```

Comment:
Why the commented out methods here?

Reply:
At first sight, those were the methods that were being used throughout all the DAO's which were using the Hibernate library. I commented those out to get an overview of where they were being used, but I forgot to delete these all together when they were no longer necessary and replaced by their persistence implementation